### ISSUE TITLE
Bring the GTK asthetic

### ISSUE THREAD

**Issue by Adidiii** [2Framesgh/Ubuntu-Yaru-BD-theme#1]
I think that the first step to recreating a Yaru theme for anything is to recreate a GTK/Adwaita theme to that thing.

**Comment by 2Framesgh** [comment]
True, I have barely updated this though, never really active on GH
